### PR TITLE
HyperExpansion axis added

### LINKED ISSUE #11: Add HEXP (Hyper Expansion) Axis
"Expansion of inner and outer space of letter forms."

Fonts to be onboarded that have this axis:
* https://github.com/google/fonts/issues/3734
* https://github.com/google/fonts/issues/3203

### PR DESCRIPTION
**Pull request by RosaWagner**
Closes #11 

For the fallbacks, I don't think it is necessarily what we shall do, but these are the current fallback for Readex Pro and Lexend (although not sure they will follow the exact same value).

### CODE REVIEW

**Comment by vv-monsalve** [comment]
> (although not sure they will follow the exact same value).

This, among other reasons, is why we are currently not registering axis with fallback positions. Only declaring Min to Max range values plus the required `Default` fallback.

Solving the per family named instances is part of the future solutions needed but so far out of the scope when onboarding the current projects.

cc @evanwadams, @davelab6

**Comment by RosaWagner** [comment]
Yep, I am expecting that too. 

For Min and Max, I guess the decision that has to be taken is if it follows the already used style names (Deca/Zetta) or something else.

File: Lib/axisregistry/data/hyper_expansion.textproto [hunk 1]
```diff
@@ -0,0 +1,12 @@
+tag: "HEXP"
+display_name: "Hyper Expansion"
+min_value: 0.0
+default_value: 0.0
+max_value: 100.0
+precision: 1
```
Comment (chrissimpkins) [inline Lib/axisregistry/data/hyper_expansion.textproto:6]: Should this be `-1`?

File: Lib/axisregistry/data/hyper_expansion.textproto [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+fallback {
+  name: "Default"
+  value: 0.0
+}
+fallback_only: false
+description: "Expansion of inner and outer space of glyphs."
```
Comment (chrissimpkins) [inline Lib/axisregistry/data/hyper_expansion.textproto:12]: Per Viv: we should expand on definition to describe how this will be distinct from space axis

**Review by chrissimpkins — CHANGES_REQUESTED** [review]
Requests inline

File: Lib/axisregistry/data/hyper_expansion.textproto (hunk 2 above)
Comment (chrissimpkins) [reply Lib/axisregistry/data/hyper_expansion.textproto:12]: OK to address this in sandbox post-merge if necesssary

**Comment by chrissimpkins** [comment]
git subtree pulled into google/fonts in https://github.com/google/fonts/pull/5301

**Pull request merged**